todo_app/templates/index.html

```diff
@@ -15,27 +15,82 @@ <h2>Items</h2>
       <table>
       <tr>
         <th>Title</th>
+        <th>Description</th>
         <th>Status</th>
+        <th>Due Date</th>
         <th>Action</th>
         <th>Delete</th>
       </tr>
           {% for item in listOfItems %}
             <tr>
-                <td>{{ item.title }}</td>
+                <td>{{ item.name }}</td>
                 <td>
-                    {% if item.status == 'Not Started' %}
+                    {% if item.description != "" %}
```

Comment:
For this falsy check, you don't need a comparator:

```suggestion
                    {% if item.description %}
```